### ISSUE TITLE
Apps on monitor 2 do not behave as expected when monitor 2 isn't detected on launch

### ISSUE THREAD

**Issue by plante-msft** [JaneaSystems/PowerToys-DevProjects#65]
Display setup:

![Image](https://github.com/user-attachments/assets/696ff8b5-59f1-4ea9-8ba7-4ca331233bd8)

Project layout:

![Image](https://github.com/user-attachments/assets/de4bab4c-3944-45ab-ab34-3cd724812ca5)

**Expected:** When the top widescreen is removed, Outlook, VS Code and Edge should be launched as minimized on Monitor 1. On restore, they will restore to maximized fullscreen. 

**Actual:** The app windows are rendered in some scaled dimensions on monitor 1. 

![Image](https://github.com/user-attachments/assets/8af71d2b-aaed-4e2c-948f-2cd46b9abea0)

**Comment by plante-msft** [comment]
@donlaci quick question - were we able to address this and launch the apps to minimized or did we run into issues there? I know launching apps to minimized states can be inconsistent.

**Comment by donlaci** [comment]
@plante-msft Yes, all apps are launched minimized if their originally saved monitor cannot be found at launch. There is the same effect as for all minimized launches: the app pops up for a short time before it gets minimized.
